src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -168,7 +168,18 @@ def _append_uid(self, user):
         return user
 
     def list(self):
-        return self.proxy('GET', 'metadata/user')
+        users = []
+        marker = None
+        while True:
```

Comment:
Can we make sure, this is not going into an endless loop?

Reply:
Guess my biggest fear is we will end up always use the same marker over and over again.